### PR TITLE
Update to cl 19.41

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by h-vetinari** [comment]
Ah right, we haven't reacted to the new vc version here yet.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -22,31 +22,38 @@ channel_targets:
...
 TOOLCHAIN_COMBINED:
+  - 14.41.17.11
   - 14.40.17.10
   - 14.29.16.11
 
 # header version corresponding to minor version (i.e. x in VS 15.x), see last digits of _MSC_FULL_VER in
 # https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering;
 # alternatively, download & inspect the *.vsix binaries (for details see install-pkg.sh)
 MSVC_HEADERS_VERSION:
+  - 14.40.33807
```
Comment (h-vetinari) [inline recipe/conda_build_config.yaml:56]: See https://github.com/conda-forge/clang-win-activation-feedstock/pull/44/commits/79097fad61dea6e8bce865b6b2f7869c8171762b
```suggestion
  - 14.41.34120
```

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
The errors have been reduced to only windows now:
```
In file included from C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h:12:
C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h:10:10: fatal error: 'vcruntime.h' file not found
   10 | #include <vcruntime.h>
      |          ^~~~~~~~~~~~~
1 error generated.
```

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Pull request merged**